app/presenters/ratings_presenter.rb

```diff
@@ -0,0 +1,4 @@
+class RatingsPresenter < Jsonite
+  property :rating
```

Comment:
Ah, I also noticed that this ends up getting returned as a string. I'm not sure why. I had to cast to a float with `to_f` when I was working on this. I assume that's better for the web client. Which reminds me: it'd be good to update the resource request test(s) to include some ratings.